content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,129 +2,169 @@
+   match.
 
 .. seealso::
    - :ref:`Manage users <users/add-individual>`
    - :doc:`Access rights </applications/general/users/access_rights>`
 
-Create or modify kanban stages
-==============================
+Create or modify stages
+=======================
 
-:guilabel:`Stages` are used to organize the *Helpdesk* pipeline and track the progress of tickets.
-Stages are customizable, and can be renamed to fit the needs of each team.
+*Stages* are used to organize the *Helpdesk* pipeline and track the progress of tickets. Stages are
+customizable, and can be renamed to fit the needs of each team.
 
-To view or modify *Helpdesk* stages, go to :menuselection:`Helpdesk --> Configuration --> Stages`.
+To view or modify *Helpdesk* stages, go to :menuselection:`Helpdesk app --> Configuration -->
+Stages`.
 
 .. important::
-   :ref:`Developer mode <developer-mode>` must be activated in order to access the stages menu. To
-   activate developer mode go to :menuselection:`Settings --> General Settings --> Developer Tools`
-   and click on :guilabel:`Activate the developer mode`.
+   :ref:`Developer mode <developer-mode>` must be activated to access the stages menu. To activate
+   developer mode go to :menuselection:`Settings --> General Settings --> Developer Tools` and click
+   on :guilabel:`Activate the developer mode`.
 
-The list view shows an overview of all the stages currently available in Helpdesk. They are listed
-in the order they appear in the pipeline. To change the order of the stages, use the arrow buttons
-on the left side of the list.
+The list view displays the stages currently available in *Helpdesk*. They are listed in the order
+they appear in the pipeline. To change the order of the stages, click and drag the buttons to the
+left of the stage name.
 
-.. tip::
-   Change the stage order on the kanban view by dragging and dropping individual columns.
-
-.. image:: getting_started/stages-create-new.png
+.. image:: getting_started/stages-list-buttons.png
    :align: center
-   :alt: View of the stage list page emphasizing the option to create a new stage
+   :alt: View of the stage list page emphasizing the buttons used to change the order the stages
+         appear in the list.
 
+.. tip::
+   Change the stage order on the kanban view by dragging and dropping individual columns.
 
-To create a new stage, click on the :guilabel:`New` button in the top left of the dashboard.
-Next, choose a name for the new stage, and add a description (though it is not required).
-Fill out the remaining fields following the steps below.
+To create a new stage, click on the :guilabel:`New` button at the top-left of the stage list. Next,
+choose a name for the new stage, and add a description, if desired. Fill out the remaining fields
+following the steps below.
 
-.. image:: getting_started/stage-settings.png
+.. image:: getting_started/new-stage-details.png
```

Comment:
The sequence field is only visible in dev mode, which I usually try to avoid when taking screenshots, but since this section does instruct users to turn on dev mode, I should probably include it. 👍